src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCaseNew.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Test;
+
+/**
+ * KernelTestCaseNew is the base class for tests needing a Kernel >= PHPUNIT 8.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @internal
+ */
+abstract class KernelTestCaseNew extends BaseKernelTestCase
```

Comment:
We should think about different names for these clases. Imagine that PHPUnit 9 breaks everything again ... should the new class be called then `KernelTestCaseNewNew` ? And this `new` class would then be the *"not really new, but old"*, etc.

Reply:
@javiereguiluz I did see that there is something similiar for the TestRunner which is called `TestRunnerV7`. Should we call it `KernelTestCaseV7` and `KernelTestCaseV8`.

Reply:
The class is now changed to KernelTestCaseV7 and V8